test/typescript-karma/Dashboards/Component/tests.js

```diff
@@ -535,3 +535,40 @@ test('DataGrid component with dataTable', async function (assert) {
         'DataGrid component should have a dataTable with columns.'
     );
 });
+
+test('KPI Component updating', async function (assert) {
+    const container = document.createElement('div');
```

Comment:
you can create
```
document.createElement('div', {
   id: 'container'
});
```

instead of additional assignment

Reply:
This would work if I used the `Utils` function from Highcharts, but `document.createElement` is native JS. :D